src/stactools/ephemeral/commands.py

```diff
@@ -0,0 +1,44 @@
+import click
+import logging
+
+from pystac import Item, Collection
+
+logger = logging.getLogger(__name__)
+
+
+def create_ephemeralcmd_command(cli):
+    """Creates the ephemeral-package command line utility."""
+
+    @cli.group(
+        "ephemeralcmd", short_help=("Commands for working with ephemeral-package"),
+    )
+    def ephemeralcmd():
+        pass
+
+    @ephemeralcmd.command(
+        "create-collection", short_help="Creates a STAC collection",
+    )
+    @click.argument("destination")
+    def create_collection_command(destination: str) -> Collection:
+        """Creates a STAC Collection
+
+        Args:
+            destination (str): A placeholder to use click and demonstrate arguments
+        """
+        assert destination
```

Comment:
Not sure if we need this assert. If a user works hard enough to pass in an empty string through click, we should let it blow up later when trying to save (or something)?